contracts/event/MultipleResultsEvent.sol

```diff
@@ -411,12 +365,10 @@ contract MultipleResultsEvent is NRC223Receiver, Ownable {
         require(value > 0, "Bet amount should be > 0");
 
```

Comment:
Actually the over paid issue should be handle here
```
if (_currentVotingRoundTotals[resultIndex].add(value) > _eventRounds[_currentRound].consensusThreshold) {
    unit diff = _betRoundTotals[resultIndex].add(value) - threshold;
    value = value.sub(diff);
    IFactory.transfer(from, diff);
}
````

Reply:
this is incorrect. betting is for round 0. there is no such thing as betting over the threshold. the result setter setting the result on L398 already makes sure they cant send more NBOT than the threshold amount.